tdvt/tdvt/tabquery.py

```diff
@@ -90,11 +90,15 @@ def build_tabquery_command_line(self, work):
 
 def tabquerycli_exists():
     global tab_cli_exe
+    tabquery_envvar = os.environ.get('TABQUERY_CLI_PATH')
+
+    if os.path.isfile(tabquery_envvar):
+        logging.debug("Found tabquery at [{0}]".format(tabquery_envvar))
```

Comment:
Maybe log that we grabbed this from the environment variable?  Just in case someone has strange errors because they forgot they set it.

Reply:
good idea, updated to add that.

also added the part that was missing (actually _setting_ tabquerycli) with logging there, too.